<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGConnectionEditorWidget</class>
 <widget class="QWidget" name="GGConnectionEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="GGMouseLabel" name="lblNumber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(148, 148, 148);</string>
     </property>
     <property name="text">
      <string notr="true">#1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtCaption"/>
   </item>
   <item>
    <widget class="GGActionEditorWidget" name="wgtAction" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lblConnectedPage">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnActivate">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/connect</normaloff>:/icons/connect</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDelete">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/delete</normaloff>:/icons/delete</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GGActionEditorWidget</class>
   <extends>QWidget</extends>
   <header>ui/components/ggactioneditorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GGMouseLabel</class>
   <extends>QLabel</extends>
   <header>ui/basic/ggmouselabel.h</header>
   <slots>
    <signal>hoverEnter()</signal>
    <signal>hoverLeave()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>txtCaption</sender>
   <signal>editingFinished()</signal>
   <receiver>GGConnectionEditorWidget</receiver>
   <slot>captionEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnActivate</sender>
   <signal>clicked()</signal>
   <receiver>GGConnectionEditorWidget</receiver>
   <slot>activateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnActivate</sender>
   <signal>toggled(bool)</signal>
   <receiver>GGConnectionEditorWidget</receiver>
   <slot>activateToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>GGConnectionEditorWidget</receiver>
   <slot>deleteMe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>activateClicked()</slot>
  <slot>activateToggled(bool)</slot>
  <slot>captionEdited()</slot>
  <slot>deleteMe()</slot>
 </slots>
</ui>
